Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -162,6 +209,55 @@ public Task<RecurringWebinar> CreateRecurringWebinarAsync(string userId, string
 				.AsObject<RecurringWebinar>();
 		}
 
+		/// <summary>
+		/// Updates an existing recurring webinar for a user.
+		/// </summary>
+		/// <param name="webinarId">The webinar ID.</param>
+		/// <param name="topic">Webinar topic.</param>
+		/// <param name="agenda">Webinar description.</param>
+		/// <param name="start">Webinar start time.</param>
+		/// <param name="duration">Webinar duration (minutes).</param>
+		/// <param name="recurrence">Recurrence information.</param>
+		/// <param name="password">Password to join the webinar. By default the password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters. This can be updated within Zoom account settings.</param>
+		/// <param name="settings">Webinar settings.</param>
+		/// <param name="trackingFields">Tracking fields.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The async task.
+		/// Zoom will return an HTTP 200 response when there is no webinar subscription present.
```

Comment:
I would move this comment right above the section of code where we check the `StatusCode` to clearly explain why we are checking for `StatusCode.Ok` but I don't think this comment needs to be in the library's public XML documentation since we are automatically handling the HTTP 200 response.

Reply:
Removed in next PR